docs/cloud/data-stores/quick-start.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 1
+title: Quick Start
+label: Cloud
+--- 
+
+
+## Getting Started with Dragonfly Cloud
+
+A Dragonfly Cloud data store represents a Redis Protocol (RESP) endpoint. To create a data store, on the Data Store tab click +Data Store. 
+
+The minimum configuration consists of a name, cloud provider, cloud region and plan. Note that the cloud provider can not be modified once the data store is created.
+
+
+![datastore](../../../static/img/add-datastore.png)
```

Comment:
This doesn't look good. Can you make it smaller and inline with the text?

Reply:
should be fixed now

Reply:
docs should line up now